scan_rr.go

```diff
@@ -1696,3 +1696,40 @@ func (rr *TKEY) parse(c *zlexer, o string) *ParseError {
 
 	return nil
 }
+
+func (rr *SVCB) parse(c *zlexer, o string) *ParseError {
+	l, _ := c.Next()
+	i, e := strconv.ParseUint(l.token, 10, 16)
+	if e != nil || l.err {
+		return &ParseError{"", "bad SVCB Priority", l}
+	}
+	rr.Priority = uint16(i)
+
+	c.Next()        // zBlank
+	l, _ = c.Next() // zString
+	rr.Target = l.token
+
+	name, nameOk := toAbsoluteName(l.token, o)
+	if l.err || !nameOk {
+		return &ParseError{"", "bad SVCB Target", l}
+	}
+	rr.Target = name
+
+	// Values (if any)
+	l, _ = c.Next()
+	var xs []string
+	for l.value != zNewline && l.value != zEOF {
+		switch l.value {
+		case zString:
+			xs = append(xs, l.token)
```

Comment:
I think these values are always quoted? Not sure if the quotation is part of the value or not

Reply:
They're of the form `a=b` or `a="b"`. Because the values of key=value pairs consist of

> basic-visible [which] is VCHAR minus DQUOTE, ";", and "\"

So I should add code to remove the double quotes if they exist.